src/contexts/account/useWithdrawals.ts

```diff
@@ -12,31 +12,30 @@ export interface PendingWithdrawal {
 const botActionInterval = 180 * 1000;
 
 export const useWithdrawalBot = (address: string | null) => {
-  const { api } = useAPI();
-  const { managerContract, accountContract } = useBlockchain();
-
-  const finalizeWithdrawal = useCallback(async () => {
-    if (!address) return;
-    const [activeGroups, deprecatedGroups] = await Promise.all([
-      managerContract.methods.getGroups().call(),
-      managerContract.methods.getDeprecatedGroups().call(),
-    ]);
-    const groups = [...activeGroups, ...deprecatedGroups];
-    for (const group of groups) {
-      const scheduledWithdrawals = await accountContract.methods
-        .scheduledWithdrawalsForGroupAndBeneficiary(group, address)
-        .call();
-      if (scheduledWithdrawals !== '0') return api.withdraw(address);
-    }
-  }, [address, managerContract, accountContract, api]);
-
-  useEffect(() => {
-    void finalizeWithdrawal();
-    const intervalId = setInterval(finalizeWithdrawal, botActionInterval);
-    return () => {
-      clearInterval(intervalId);
-    };
-  }, [finalizeWithdrawal]);
+  // const { api } = useAPI();
+  // const { managerContract, accountContract } = useBlockchain();
+  // const finalizeWithdrawal = useCallback(async () => {
+  //   if (!address || !managerContract || !accountContract) return;
+  //   const [activeGroups, deprecatedGroups] = await Promise.all([
+  //     // TODO find replacement as this is removed from v2
+  //     managerContract.methods.getGroups().call(),
+  //     managerContract.methods.getDeprecatedGroups().call(),
+  //   ]);
+  //   const groups = [...activeGroups, ...deprecatedGroups];
+  //   for (const group of groups) {
+  //     const scheduledWithdrawals = await accountContract.methods
+  //       .scheduledWithdrawalsForGroupAndBeneficiary(group, address)
+  //       .call();
+  //     if (scheduledWithdrawals !== '0') return api.withdraw(address);
+  //   }
+  // }, [address, managerContract, accountContract, api]);
+  // useEffect(() => {
+  //   void finalizeWithdrawal();
+  //   const intervalId = setInterval(finalizeWithdrawal, botActionInterval);
+  //   return () => {
+  //     clearInterval(intervalId);
+  //   };
+  // }, [finalizeWithdrawal]);
```

Comment:
I commented this out for now because manager no longer has getGroups functions. I will add back before it is merged into prod